respeaker_ros/launch/sample_respeaker.launch

```diff
@@ -13,14 +13,20 @@
   <arg name="language" default="en-US"/>
   <!-- self cancellation -->
   <arg name="self_cancellation" default="true"/>
+  <!-- audio info topic name -->
+  <arg name="audio_info" default="audio_info"/>
 
   <node if="$(arg publish_tf)"
         name="static_transformer" pkg="tf" type="static_transform_publisher"
         args="0 0 0 0 0 0 1 map respeaker_base 100"/>
 
   <node if="$(arg launch_respeaker)"
         name="respeaker_node" pkg="respeaker_ros" type="respeaker_node.py"
-        respawn="true" respawn_delay="10" />
+        respawn="true" respawn_delay="10" >
+    <rosparam subst_value="true" >
+      publish_multichannel: $(arg publish_multichannel)
```

Comment:
You may forget to add
```
<arg name="publish_multichannel" default="false"/>
```

This type of error cannot be detected by GitHub Action.
This is because in GitHub Action we cannot use respeaker and we do not run `sample_respeaker.launch`

My suggestion is to use `roslaunch_add_file_check()` in `if(CATKIN_ENABLE_TESTING)` section.
```
  find_package(roslaunch REQUIRED)
  roslaunch_add_file_check(launch/sample_respeaker.launch)
```

When I use this check to the current `sample_respeaker.launch`, I get the following error.
```
FAILURE:
[/home/naoya/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/respeaker_ros/launch/sample_respeaker.launch]:
	[/home/naoya/ros/melodic/src/jsk-ros-pkg/jsk_3rdparty/respeaker_ros/launch/sample_respeaker.launch] requires the 'publish_multichannel' arg to be set
```

c.f. https://github.com/jsk-ros-pkg/jsk_robot/blob/eac815173b8f0e0df3a1f230dd9b68ec937fddfe/jsk_fetch_robot/jsk_fetch_startup/CMakeLists.txt#L78

Reply:
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/350/commits/305343a81a5bc9698443a5c47c32ce8a22b1a3c0
Thanks for your comment.
I added the test code.